pkg/webhooks/clusterqueue_webhook.go

```diff
@@ -144,6 +145,15 @@ func validatePreemption(preemption *kueue.ClusterQueuePreemption, path *field.Pa
 	return allErrs
 }
 
+func validateCQAdmissionChecks(spec *kueue.ClusterQueueSpec, path *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+	if len(spec.AdmissionCheckStrategy) != 0 && len(spec.AdmissionChecks) != 0 {
+		allErrs = append(allErrs, field.Invalid(path, spec, "Either AdmissionChecks or AdmissionCheckStrategy can be set, but not both"))
```

Comment:
```suggestion
		allErrs = append(allErrs, field.Invalid(path, spec, "must have either admissionChecks or admissionCheckStrategy"))
```
The field.Invalid() adds the path as a prefix.